src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -640,10 +640,14 @@ private void Dispose(bool disposing)
                 // This is a special case for 1 version of Nuget.Frameworks that was shipped with using identifier NET5 instead of NETCoreApp5 for .NET 5.
                 || chosenFramework.Name.IndexOf("net5", StringComparison.OrdinalIgnoreCase) >= 0)
             {
-#if NETCOREAPP
-                // We are running in vstest.console that is either started via dotnet.exe
-                // or via vstest.console.exe .NET Core executable. For AnyCPU dlls this
-                // should resolve 32-bit SDK when running from 32-bit dotnet process and
+                // We are running in vstest.console that is either started via dotnet
+                // or via vstest.console.exe. The architecture of the current process
+                // determines the default architecture to use for AnyCPU dlls
+                // and other sources that don't dictate architecture (e.g. js files).
```

Comment:
Out of curiosity, do we run something for javascript?

Reply:
there is chuzpath adapter. https://marketplace.visualstudio.com/items?itemName=vs-publisher-2795.ChutzpahTestAdapterfortheTestExplorer